src/Arcus.Messaging.Tests.Integration/MessagePump/ServiceBusMessagePump.ResiliencyTests.cs

```diff
@@ -136,90 +138,6 @@ private async Task<TemporaryTopicSubscription> CreateTopicSubscriptionForMessage
                 rule: new CreateRuleOptions("MessageId", new SqlRuleFilter($"sys.messageid in ({string.Join(", ", messages.Select(m => $"'{m.MessageId}'"))})")));
         }
 
-        [Fact]
-        public async Task ServiceBusTopicMessagePump_PauseViaCircuitBreaker_RestartsAgainWithOneMessage()
```

Comment:
Removed this test as it was mostly testing the wait time, not the circuit breaker functionality. Was useful during development, but unnecessary for the test suite as we have the `ServiceBusMessageQueuePump_WithUnavailableDependencySystem_CircuitBreaksUntilDependencyBecomesAvailable` and like tests that more check the circuit breaker functionality as a whole.

Reply:
I think this test is valuable because it shows / verifies that the configured wait-times are respected ?

Reply:
Yes, it shows that if we set a wait time, the system correctly uses the wait time: it is a complex get/set integration test that verifies little and has a bit setup cost. And what's more: the implementation can easily circumvent the assertion here by waiting on another point.
It was useful in the beginning as I used it during the initial test-driven development, but now, it feels out of touch with the circuit breaker functionality.